src/components/ArticleList.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import { fetchAllArticles } from "../api";
+import HourglassBottomTwoToneIcon from "@mui/icons-material/HourglassBottomTwoTone";
+
+const ArticleList = ({ articles, setArticles }) => {
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    fetchAllArticles(setArticles).then((articleList) => {
```

Comment:
unsure why setArticles is being passed into the fetch here.

Reply:
ammended